### PR TITLE
Static Code Climate examples

### PR DESCRIPTION
**Pull request by PyvesB**
We've hit a wall in #1518, our Code Climate badges have been unreliable for months now.

This pull request aims at sparing some of our limited API quota by switching to static examples on the homepage.

I'm not willing to migrate the badges to the new service architecture at this point as we've already put in quite a lot of investigation and development effort in these badges. The least we can say is that @CodeClimate have not been very cooperative and have been dragging their feet since one of our users first contacted them back in April.

### CODE REVIEW

**Comment by PyvesB** [comment]
I Have deployed a [review app](https://shields-staging-pr-2201.herokuapp.com), everything seems fine apart from the static examples not shwoing up. Am I being thick? 😄

**Comment by chris48s** [comment]
I think if you want to use `staticExample` in `all-badge-examples.js`, you need to define it as a static badge URL instead of just a javascript object because its not going though any of this:

https://github.com/badges/shields/blob/95f1b50194a58f4fa11c235822442573005729e0/services/base.js#L90-L106

I've not checked if this works, but given `LegacyService` extends `BaseService`, you can probably move the examples out of `all-badge-examples.js` and into the service class without having to do a full refactor of the service class. You might also need to define `url()` in there so you've got a base..

**Comment by PyvesB** [comment]
@chris48s thank you for your help! This piece of work should now be ready for review.

File: services/codeclimate/codeclimate.service.js [hunk 1]
```diff
@@ -134,4 +134,67 @@ module.exports = class Codeclimate extends LegacyService {
...
+        },
+      },
+      {
+        title: 'Code Climate coverage',
+        exampleUrl: 'coverage/jekyll/jekyll',
+        urlPattern: 'coverage/:userRepo',
+        staticExample: {
+          label: 'coverage',
+          message: '95%',
+          color: 'yellowgreen',
```
Comment (chris48s) [inline services/codeclimate/codeclimate.service.js]: This should be green (or a lower value)

File: services/codeclimate/codeclimate.service.js [hunk 2]
```diff
@@ -134,4 +134,67 @@ module.exports = class Codeclimate extends LegacyService {
...
+        },
+      },
+      {
+        title: 'Code Climate coverage (letter)',
+        exampleUrl: 'coverage-letter/jekyll/jekyll',
+        urlPattern: 'coverage-letter/:userRepo',
+        staticExample: { label: 'coverage', message: 'A', color: 'green' },
```
Comment (chris48s) [inline services/codeclimate/codeclimate.service.js]: this should be brightgreen (or a lower value)

File: services/codeclimate/codeclimate.service.js [hunk 3]
```diff
@@ -134,4 +134,67 @@ module.exports = class Codeclimate extends LegacyService {
...
+      },
+      {
+        title: 'Code Climate technical debt',
+        exampleUrl: 'tech-debt/jekyll/jekyll',
+        urlPattern: 'tech-debt/:userRepo',
+        staticExample: {
+          label: 'technical debt',
+          message: '3%',
+          color: 'green',
```
Comment (chris48s) [inline services/codeclimate/codeclimate.service.js]: this should also be brightgreen (or a lower value)

**Comment by PyvesB** [comment]
You're right, I got confused with the shades of green!

**Comment by RedSparr0w** [comment]
Nice work, hopefully this reduces the API hits a fair amount.

**Pull request merged**